components/gallery/GalleryItem.vue

```diff
@@ -137,7 +138,9 @@ const showCongratsMessage = ref(false)
 const onNFTBought = () => {
   activeTab.value = tabs.activity
   showCongratsMessage.value = true
+  tabsRef.value?.refreshActivityTab()
```

Comment:
Do refetch after buying action may be not the best solution. It's still possible to fetch the old events data.